python/cuml/cuml_accel_tests/test_core.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+#
```

Comment:
This file contains tests for `cuml.accel` (but not estimator specific). In the future we should expand this to better test `install`/`magics`/... as needed.

Reply:
Would this file include tests for click CLI or shoudl we do a CLI test?

Reply:
I personally like to split tests for different bits of functionality up between files, but no strong thoughts. Mostly was trying to distinguish `test_accelerator.py` (which is not cuml-specific accelerator tests) with `test_core.py` (which is).